encryptedlogging/src/androidTest/kotlin/com/automattic/encryptedlogging/release/ReleaseStack_EncryptedLogTest.kt

```diff
@@ -61,93 +63,121 @@ internal class ReleaseStack_EncryptedLogTest {
 
     @Before
     fun setUp() {
-        val context = InstrumentationRegistry.getInstrumentation().context
-        val preferenceUtilsWrapper = PreferenceUtils.PreferenceUtilsWrapper(context)
-        cleanSharedPreferencesState(preferenceUtilsWrapper)
-        initializeEncryptedLogStore(context, preferenceUtilsWrapper)
-        WellSql.delete(EncryptedLogModel::class.java).execute()
-        mDispatcher.register(this)
+        Dispatchers.setMain(testDispatcher)
```

Comment:
What's the reason for modifying the main dispatcher? I think this SDK shouldn't do anything on the main thread, so this looks redundant. I propose removing it, here's my idea: c686859302f7aab7c8b05602ea576c33fd058c6a. Tests passed.

Reply:
Oh true, you're totally right, I guess this a habit of working on an Android codebase and writing tests for `ViewModel` classes, plus AI suggesting it, done: 3eac8a20f3d32ee16f3e512dcb9a3840636a57c5